alf/trainers/policy_trainer.py

```diff
@@ -331,12 +334,13 @@ def play(root_dir,
 
         episode_reward += float(time_step.reward)
 
-        if time_step.is_last():
+        if time_step.is_last() or episode_length >= max_episode_length > 0:
             logging.info("episode_length=%s episode_reward=%s" %
                          (episode_length, episode_reward))
             episode_reward = 0.
             episode_length = 0.
             episodes += 1
+            time_step = env.reset()
```

Comment:
seems no need to return time_step

Reply:
in case it ends because of max_episode_length, the previous time_step is not correct.

Reply:
I mean reset() should be called but time_step will always be overwritten.

Reply:
I don't understand what you mean.

Reply:
Sorry I misunderstood the code